### PR TITLE
docs: how to read the graph

### PR DESCRIPTION
**Pull request by Eomm**
I'm not sure if I should write more content or it is fine to let the user approaches this visualization

### CODE REVIEW

File: example/app.js [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
-module.exports = async function (fastify, opts) {
+async function app(fastify, opts) {
   await fastify.register(require('fastify-overview'), {
     addSource: true
   })
   await fastify.register(require(path.join(__dirname, '../')))
 
+  fastify.addHook('onRequest', function globalHook(request, reply, done) {
+    done()
+  })
```
Comment (simoneb) [inline example/app.js:14]: let's use modern syntax! async/await

File: example/app.js [hunk 2]
```diff
@@ -19,3 +23,6 @@ module.exports = async function (fastify, opts) {
     options: Object.assign({}, opts)
   })
 }
+
+app[Symbol.for('fastify.display-name')] = 'App Name'
```
Comment (simoneb) [inline example/app.js:27]: what's this?

**Review by simoneb — APPROVED** [review]
LGTM, couple of comments

File: example/app.js (hunk 2 above)
Comment (Eomm) [reply example/app.js:27]: `fastify-overview` uses [`fastifyInstance.pluginName`](https://www.fastify.io/docs/latest/Reference/Server/#pluginname) under the hood.

That symbol defines the context's name, replacing the file name.

This example benefits a more preciserer visualization thanks to wording nodes.

File: example/app.js (hunk 1 above)
Comment (Eomm) [reply example/app.js]: done

File: example/app.js (hunk 2 above)
Comment (simoneb) [reply example/app.js:27]: shall we not use a public API to define this? it's weird to rely on such internals

Comment (Eomm) [reply example/app.js:27]: It is used by the `fastify-plugin` https://github.com/fastify/fastify-plugin/blob/3a04d1a868c1030cfe8b32e72292a12271672b08/plugin.js#L48

It should be documented here as the `skip-override` one:
https://www.fastify.io/docs/latest/Reference/Plugins/#handle-the-scope

I will fix it on the fastify repo

Comment (simoneb) [reply example/app.js:27]: sounds good thanks

Comment (Eomm) [reply example/app.js:27]: Done here https://github.com/fastify/fastify/pull/4191

**Review by simoneb — COMMENTED** [review]
I'm happy with this 🚀

**Comment by Eomm** [comment]
Would you mind merging it?
I don't have the grants to do it.

**Pull request merged**